src/DetailsView/components/start-over-dropdown.tsx

```diff
@@ -54,26 +57,28 @@ export class StartOverDropdown extends React.Component<StartOverProps, StartOver
             return null;
         }
 
-        const { testName } = this.props;
+        return <ContextualMenu onDismiss={() => this.dismissDropdown()} target={this.state.target} items={this.getMenuItems()} />;
+    }
 
-        return (
-            <ContextualMenu
-                onDismiss={() => this.dismissDropdown()}
-                target={this.state.target}
-                items={[
-                    {
-                        key: 'assessment',
-                        name: 'Start over Assessment',
-                        onClick: this.onStartOverAllTestsMenu,
-                    },
-                    {
-                        key: 'test',
-                        name: `Start over ${testName}`,
-                        onClick: this.onStartOverTestMenu,
-                    },
-                ]}
-            />
-        );
+    private getMenuItems(): IContextualMenuItem[] {
+        const { testName, detailsViewRightContentPanel } = this.props;
+        const items: IContextualMenuItem[] = [
+            {
+                key: 'assessment',
+                name: 'Start over Assessment',
+                onClick: this.onStartOverAllTestsMenu,
+            },
+        ];
+
+        if (detailsViewRightContentPanel !== 'Overview') {
```

Comment:
Is 'Overview' part of a type or enum??? I have the feeling having this values hardcoded here to be part of the logic not to be a good idea in general (`VisualizationType` comes to mind)

Reply:
yep. moved that logic to right panel config. As suggested by Wahaj offline.